package/MDAnalysis/analysis/base.py

```diff
@@ -126,9 +126,14 @@ def _setup_frames(self, trajectory, start=None, stop=None, step=None):
         """
         self._trajectory = trajectory
         start, stop, step = trajectory.check_slice_indices(start, stop, step)
-        self.start = start
-        self.stop = stop
-        self.step = step
+        if start is not None:
+            self._start = start
+        elif stop is not None:
+            self._stop = stop
+        elif step is not None:
+            self._step = step
+        else:
+            warnings.warn("Deprecation Warning", DeprecationWarning)
```

Comment:
Aside from the above comment, I'm not sure I understand how this relates to setting verbose. Apologies if I'm just missing something simple, any chance you could provide extra information on why you're doing this?